glitch/apps.py

```diff
@@ -16,30 +16,52 @@
 )
 
 
-def glitch_image(input_path: str, output_path: str) -> None:
+def glitch_image(input_path: str, output_path: str, options: dict) -> None:
     """ swaps some random blocks, random moves channels and adds salt and pepper noise to the image
     """
     image = imageio.imread(input_path)
 
+    block_movement    = float(options['block_movement'])
+    block_size        = float(options['block_size'])
+    noise_intensity   = float(options['noise_intensity'])
+    noise_amount      = float(options['noise_amount'])
+    channels_movement = float(options['channels_movement'])
+    
+    max_blocksize = {
+        'tall': (int(block_size * 300), int(800 * block_size)),
+        'wide': (int(block_size * 400), int(100 * block_size))
```

Comment:
lets stick to standard `witdh` and `height`

Reply:
This was just an attempt at being descriptive, since I think the purpose was to have tall blocks (width < height) and wide, fat ones (width > height) when swapping them.

How would you refactor this so it is descriptive and standard enough?